Misc/NEWS.d/next/Library/2017-11-05-11-42-56.bpo-25942.o5lBhR.rst

```diff
@@ -0,0 +1 @@
+Added new parameter, ``cleanup_timeout``, to ``subprocess.run()``
```

Comment:
kill_timeout may be more clear, the time we give child process to terminate before we kill it.

Reply:
I am ambivalent regarding the parameter name.